backend/calculate_streamer.js

```diff
@@ -219,36 +204,37 @@ function matchStreamers(prefs, streamers) {
     };
 
     // check against average viewers preference
-    if (streamer) {
+    if (streamer.avg_viewers) {
       let viewerRange = getMinMaxViewers(prefs.average_viewers);
       let score = countNearScore(
-        ATTRIBUTE_POINTS.average_viewers,
+        RANK_SCORE.average_viewers,
         streamer.avg_viewers,
         viewerRange[0],
         viewerRange[1],
         SCORE_NEAR_THRESHOLD.avg_viewers
       );
       scores += score;
-      stats[streamer.id]["Viewers"] = Math.ceil(
-        (score / ATTRIBUTE_POINTS.average_viewers) * 100
+      stats[streamer.id]["Viewers"] = Math.round(
```

Comment:
using round to fix floating rounding issue ex: 101% would be shown because the decimals are not completely zero